tests/autorestart_test/test_container_autorestart.py

```diff
@@ -213,8 +213,9 @@ def test_containers_autorestart(duthost):
     disabled_containers = get_disabled_container_list(duthost)
 
     for container_name in container_autorestart_states.keys():
-        # Skip testing the database container or containers/services which are disabled
-        if container_name in disabled_containers or container_name == "database":
+        # Skip testing the database container, radv container on T1 devices and containers/services which are disabled
+        if (container_name in disabled_containers or container_name == "database"
+                or (container_name == "radv" and tbinfo["topo"]["name"] == "t1")):
```

Comment:
@yxieca: What about topologies like "t1_lag"? Is there a better way to determine it is a T1 topology?

Reply:
@jleveque In this PR https://github.com/Azure/sonic-mgmt/pull/2217, at the line 65 of test_acl.py, it is used to determine whether the device is T1 or not.

Reply:
That is a perfect example of my point. There are multiple T1-based topologies. You're using `tbinfo["topo"]["name"] == "t1"`, but it will not catch them all. I was wondering if there is a convenience method for catching all, or if it would be safe to use `startswith` rather than `==`.

Reply:
Right now, I adopted the usage in the PR #2217, we can talk to Ying to confirm that the `startswith` method is convenient and safe one.

Reply:
@yxieca: Can you answer my question above, "is a convenience method for catching all, or if it would be safe to use startswith rather than =="?

Reply:
use 'type' instead. example: https://github.com/Azure/sonic-mgmt/blob/master/tests/iface_namingmode/test_iface_namingmode.py#L491